app/views/tournaments/show.html.slim

```diff
@@ -36,8 +36,8 @@
                   .badge.badge-secondary NO
               - if @tournament.allow_streaming_opt_out?
                 li.list-group-item
-                  .small.text-secondary Streaming opt out:
-                  - if @current_user_player.streaming_opt_out?
+                  .small.text-secondary Swiss games can be streamed:
```

Comment:
is this strictly for Swiss or generally for the tournament?

Reply:
Specifically for Swiss. The idea is it's unlikely we'll be able to exclude people from streaming in the cut. I'm not 100% sure how clean cut it should be though.